### PR TITLE
Inverse of an array

### PR DESCRIPTION
**Pull request by Op-panda**
#847

### CODE REVIEW

**Review by krishna-NIT — CHANGES_REQUESTED** [review]
Resolve Conflict

**Comment by Op-panda** [comment]
should i delete that ??

**Comment by Op-panda** [comment]
@raksha009 @nakul-19
@tarun26091999 @rudrakshi99 @plazzy99 @nakul-19

File: Code/C++/inverse_of_an_array.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Input:
+1)First line contains t, number of test cases.
+2)Second line contains size of the original array.
+3)Third line contains elements of the array.
+
+Output:
+1)contains t lines of output, inverse array of each original array. 
+
+Sample input:
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:24]: Hey, plz do not add sample test cases and complexity here.  Only provide a brief description of problem statement here.

File: Code/C++/inverse_of_an_array.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+1 4 2 3 0
+0 1 2
+
+time comlexity - O(n) */
+
+
+#include <bits/stdc++.h>
+using namespace std;
+
+// ALGORITHM -->
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:41]: Add algorithm in problem description part

File: Code/C++/inverse_of_an_array.cpp [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+// create two arrays, ori and inv.
+// give input for ori array.
+// create a temporary variable then, and store the element from the original array.
+// In inv array for each value of temp, store the index of ori array.
+// Display the inverse array.
+
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:47]: Add in 1 line above this function about its role.

File: Code/C++/inverse_of_an_array.cpp [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        int temp = a[i];
+        b[temp] = i;
+    }
+    for (int i = 0; i < n; i++)
+        cout << b[i] << " ";
+}
+
+int main()
+{
+    // Driver code begins
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:61]: Remove this comment.

File: Code/C++/inverse_of_an_array.cpp [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    int t, n;
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:62]: cout what is n, t etc. Take appropriate variable names.

File: Code/C++/inverse_of_an_array.cpp [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        int inv[n];
+
+        for (int i = 0; i < n; i++)
+            cin >> ori[i];
+
+        inverse(ori, n, inv);
+    }
+    return 0;
+}
+
```
Comment (raksha009) [inline Code/C++/inverse_of_an_array.cpp:77]: Add testcases and complexities here. Plz follow guidelines,

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by Op-panda** [comment]
all changes done I hope

**Pull request merged**